src/ssl.c

```diff
@@ -16941,15 +16944,80 @@ int wolfSSL_CTX_set_min_proto_version(WOLFSSL_CTX* ctx, int version)
     return CheckSslMethodVersion(ctx->method->version.major, ctx->mask);
 }
 
+/* number of protocol versions listed in table */
+#define NUMBER_OF_PROTOCOLS 7
+
+/* Sets the min protocol version allowed with WOLFSSL_CTX
+ * returns WOLFSSL_SUCCESS on success */
+int wolfSSL_CTX_set_min_proto_version(WOLFSSL_CTX* ctx, int version)
+{
+    const int verTbl[] = {SSL3_VERSION, TLS1_VERSION, TLS1_1_VERSION,
```

Comment:
Since this table is repeated three or four times, make this file global. You can redefine `NUMBER_OF_PROTOCOLS` as `sizeof(verTbl)/sizeof(int)`.

Reply:
Made  the version table global and each function refer it. Also redefined NUMBER_OF_PROTOCOLS as you suggested.